### PR TITLE
Update live stream description for progress markers user case

### PR DESCRIPTION
**Pull request by cacheung**
Update live stream description for progress markers user case
Updated for Chromecast and JS3.x

### CODE REVIEW

File: reference/chromecast/ADBMobile.media.html [hunk 1]
```diff
@@ -1582,7 +1582,7 @@ <h4 class="name" id=".setDelegate"><span class="type-signature">(static) </span>
     <p>Sets an instance of MediaDelegate to be used by the ADBMobile library.<br>
 The delegate object passed should have 2 functions.</p>
 <ol>
-<li><code>getCurrentPlaybackTime()</code> - Returns current playhead in seconds. For video-on-demand (VOD), the value is specified in seconds from the beginning of the media item. For live streaming, the value is specified as the number of seconds since midnight UTC on that day.</li>
+<li><code>getCurrentPlaybackTime()</code> - Returns current playhead in seconds. For video-on-demand (VOD), the value is specified in seconds from the beginning of the media item.  For live streaming, if the player does not provide information about the content duration, the value can be specified as the number of seconds since midnight UTC of that day. Note: When using progress markers, the content duration is required and the playhead needs to be updated as number of seconds from the beginning of the media item, starting with 0.</li>
```
Comment (emdobrin) [inline reference/chromecast/ADBMobile.media.html:1585]: I think we can drop the note for this getter and keep it only in the context of updatePlayhead.

**Review by emdobrin — COMMENTED** [review]
Looks good, left a small suggestion

File: reference/chromecast/ADBMobile.media.html (hunk 1 above)
Comment (cacheung) [reply reference/chromecast/ADBMobile.media.html]: Updated.

File: reference/chromecast/ADBMobile.media.html [hunk 2]
```diff
@@ -1582,7 +1582,7 @@ <h4 class="name" id=".setDelegate"><span class="type-signature">(static) </span>
     <p>Sets an instance of MediaDelegate to be used by the ADBMobile library.<br>
 The delegate object passed should have 2 functions.</p>
 <ol>
-<li><code>getCurrentPlaybackTime()</code> - Returns current playhead in seconds. For video-on-demand (VOD), the value is specified in seconds from the beginning of the media item. For live streaming, the value is specified as the number of seconds since midnight UTC on that day.</li>
+<li><code>getCurrentPlaybackTime()</code> - Returns the current playhead in seconds. For video-on-demand (VOD), the value is specified in seconds from the beginning of the media item.  For live streaming, if the player does not provide information about the content duration, the value can be specified as the number of seconds since midnight UTC of that day.</li>
```
Comment (addb) [inline reference/chromecast/ADBMobile.media.html:1585]: nit: there is an extra space before _**For** Live..._

Comment (cacheung) [reply reference/chromecast/ADBMobile.media.html]: removed the extra space.

File: reference/javascript_3x/APIReference.md [hunk 3]
```diff
@@ -429,7 +429,7 @@ ADB.Media.trackEvent(event, info, contextData);
 
 #### updatePlayhead
 
-Provide current media playhead to media tracker. For accurate tracking, call this method whenever playhead changes during playback.
+Provides a media tracker with the current media playhead. For accurate tracking, call this method multiple times when the playhead changes. If the player does not notify playhead changes, call this method once every second with the most recent playhead.
```
Comment (addb) [inline reference/javascript_3x/APIReference.md:432]: Provides the current media playhead to the media tracker instance. For accurate tracking, call this method everytime the playhead changes. If the player does not notify playhead changes, call this method once every second with the most recent playhead.

**Review by addb — APPROVED** [review]

**Pull request merged**